.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["flowtype.flow-for-vscode"]
+}
```

Comment:
This is unrelated with the PR let's remove it for now and we can discuss if it's worth it adding it:

```suggestion
```